src/libsync/clientsideencryption.cpp

```diff
@@ -1097,9 +1165,15 @@ void ClientSideEncryption::writeMnemonic(const AccountPtr &account)
     job->setInsecureFallback(false);
     job->setKey(kck);
     job->setTextData(_mnemonic);
-    connect(job, &WritePasswordJob::finished, [](Job *incoming) {
-        Q_UNUSED(incoming);
+    connect(job, &WritePasswordJob::finished, [this, account, nextCall = std::move(nextCall)](Job *incoming) mutable {
```

Comment:
@mgallien why `mutable`?

Reply:
because that is needed to pass `nextCall` by value inside the lambda